vsix/ProjectTemplates/VC/Windows Universal/BlankApp/App.h

```diff
@@ -6,7 +6,6 @@ namespace winrt::$safeprojectname$::implementation
     struct App : AppT<App>
     {
         App();
-
```

Comment:
Keep this line, it's stylistic and meant to visually distinguish the constructor from member methods